lib/display/display.ex

```diff
@@ -4,18 +4,31 @@ defmodule Inky.Display do
   """
 
   alias Inky.LookupTables
-  
+
   @type t() :: %__MODULE__{}
 
-  @enforce_keys [:type, :width, :height, :packed_dimensions, :rotation, :accent, :luts]
+  @enforce_keys [:type, :width, :height, :rotation, :accent]
   defstruct type: nil,
             width: 0,
             height: 0,
             packed_dimensions: %{},
+            packed_resolution: nil,
             rotation: 0,
             accent: :black,
             luts: <<>>
 
+  @spec spec_for(:impression) :: Inky.Display.t()
+  def spec_for(type = :impression) do
+    %__MODULE__{
+      type: type,
+      width: 600,
+      height: 448,
+      packed_resolution: <<2, 88, 1, 192>>, # I used the struct.pack in Python to generate this
```

Comment:
Some observations...

`(2*255 + 88) = 600-2 = 598`
`(1*255 + 192) = 488-1 = 447`

How do these numbers relate to whatever input you put into python? Also, if we're not going to do this properly in elixir, we should at the very least have the actual code here, not just a mention of "it was done in python".

The dimensions for the Inky were kept in a struct, why not do the same thing here and convert it to a binary when it's required?